src/cohort/cohort.module.ts

```
@@ -41,6 +41,7 @@ import { FormsModule } from 'src/forms/forms.module';
     PostgresModule,
     ElasticsearchModule,
     forwardRef(() => FormsModule),
+
```

ESLint

[error] Delete `⏎`

(prettier/prettier)